pulpcore/cli/core/repository.py

```diff
@@ -32,3 +47,49 @@ def repository(ctx: click.Context, pulp_ctx: PulpCLIContext) -> None:
 
 repository.add_command(list_command(decorators=filter_options))
 repository.add_command(version_command(decorators=[], list_only=True))
+
+
+@repository.command()
+@pass_repository_context
+@pass_pulp_context
+@repositories_option
+@pulp_option(
+    "--all",
+    "all_repos",
+    is_flag=True,
+    show_default=True,
+    default=False,
+    help=_("Select all repositories."),
+)
+@click.option(
+    "--keep-version",
+    "keep_versions",
+    type=tuple([str, int]),
+    help=_("List of (pulp hrefs) repository versions to keep."),
+    callback=_version_list_callback,
+    multiple=True,
+    required=False,
+)
+def reclaim_space(
+    pulp_ctx: PulpCLIContext,
+    repository_ctx: PulpRepositoryContext,
+    repositories: Iterable[EntityFieldDefinition],
+    all_repos: bool,
+    keep_versions: Iterable[PulpRepositoryVersionContext],
+) -> None:
+    pulp_ctx.needs_plugin(PluginRequirement("core", specifier=">=3.19.0"))
```

Comment:
This goes in the glue layer.

Reply:
@mdellweg can you elaborate please?

Reply:
The information that this feature needs a certain pulpcore version is not limited to the cli and should be asserted on all users of pulp-glue.